src/components/Card/Card.stories.mdx

```diff
@@ -221,6 +221,44 @@ prop to override the default width.
 <Canvas>
   <Story name="Card Image Size">
     <SimpleGrid columns={1}>
+      <Card
```

Comment:
These two would be good to add to the "Row Cards" section as well.

Reply:
Very true. Not sure how I missed adding those in the first place.